grakn-engine/src/main/java/ai/grakn/engine/user/SystemKeyspaceUsers.java

```diff
@@ -55,22 +55,22 @@
      */
     @Override
     public boolean addUser(Json userJson) {
-        final Var user = var().isa(USER_ENTITY);
+        final Var[] user = {var().isa(USER_ENTITY)};
```

Comment:
I thought it was because of that. I actually find this even more unreadable. I would prefer to just use the traditional `for` and `entry` rather than to start creating these things everywhere. I think this will set a bad example for the future.

That is my opinion though. I would ask you to change that but if you feel strongly about it then feel free to merge.